components/campaign/CampaignDetailView/index.tsx

```diff
@@ -267,6 +290,10 @@ class CampaignDetailView extends React.Component<Props, State> {
           title={t`Name`}
           description={campaign.name}
         />
+        <SettingsButton
```

Comment:
I might drop a link to this on the chaos bag simulator as well, just to increase the number of places it appears in context. (though I guess we can add that later since it would probably want to take Seal into account as well).

Reply:
I looked at this and while I'm not opposed, I didn't see a great spot for it.